backend/src/Designer/Services/Implementation/Organisation/CodeListService.cs

```diff
@@ -107,18 +107,22 @@ public async Task<List<Option>> UploadCodeList(string org, string repo, string d
             throw new InvalidOptionsFormatException("Uploaded file is missing one of the following attributes for an option: value or label.");
         }
 
-        var altinnAppGitRepository = _altinnGitRepositoryFactory.GetAltinnAppGitRepository(org, repo, developer);
-        await altinnAppGitRepository.CreateOrOverwriteOptionsList(codeListId, deserializedCodeList, cancellationToken);
+        AltinnOrgGitRepository altinnOrgGitRepository = _altinnGitRepositoryFactory.GetAltinnOrgGitRepository(org, repo, developer);
+        await altinnOrgGitRepository.CreateCodeList(codeListId, deserializedCodeList, cancellationToken);
 
-        return deserializedCodeList;
+        List<OptionListData> codeLists = await GetCodeLists(org, repo, developer, cancellationToken);
+        return codeLists;
     }
 
     /// <inheritdoc />
-    public void DeleteCodeList(string org, string repo, string developer, string codeListId)
+    public async Task<List<OptionListData>> DeleteCodeList(string org, string repo, string developer, string codeListId, CancellationToken cancellationToken = default)
     {
-        var altinnAppGitRepository = _altinnGitRepositoryFactory.GetAltinnAppGitRepository(org, repo, developer);
+        AltinnOrgGitRepository altinnOrgGitRepository = _altinnGitRepositoryFactory.GetAltinnOrgGitRepository(org, repo, developer);
+
+        altinnOrgGitRepository.DeleteCodeList(codeListId);
```

Comment:
I think we need this check, but not sure 🤔 
```suggestion
        cancellationToken.ThrowIfCancellationRequested();
        
        AltinnOrgGitRepository altinnOrgGitRepository = _altinnGitRepositoryFactory.GetAltinnOrgGitRepository(org, repo, developer);

        altinnOrgGitRepository.DeleteCodeList(codeListId);
```

Reply:
Ty for catching it!